hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -647,15 +657,18 @@ private void applyDelta(Blob blob, HollowReadStateEngine stateEngine) throws IOE
 
     private void validate(HollowProducer.ReadState readState) {
         ProducerStatus.Builder status = listeners.fireValidationStart(readState);
+        OverallValidationStatus.OverallValidationBuilder valStatus = OverallValidationStatus.builder().withReadState(readState).withVersion(readState.getVersion());
```

Comment:
Just wondering, if we could use a better name for this one - like CycleValidationStatus